tools/exploit/egghunter.rb

```diff
@@ -129,7 +128,6 @@ def run
         list_formats
         return
       end
-
```

Comment:
I'm not strongly against this line being deleted, but if it's just a hang over from deleting previous debug code - it's probably worth reverting

Reply:
I'll make sure to do that